lms/services/file.py

```diff
@@ -55,6 +55,23 @@ def upsert(self, file_dicts):
             update_columns=["name", "size", "updated"],
         )
 
+    def _file_search_query(
```

Comment:
Use the style we use in other services, build a query here and use it within the service.